ontology/epso.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: ontology_detail
+id: EPSO
+label: Epilepsy Ontology
+title: Epilepsy Ontology
+description: A application driven Epilepsy Ontology sith official terms from the ILAE.
+domain: disease
+tracker: https://github.com/phwegner/Epilepsyontology/issues
+dependencies:
+  - id: BFO
+products:
+  - id: epso.owl
+  - id: EPSO_merged.owl
+contact:
+  email: alpha.tom.kodamullil@scai.fraunhofer.de
+  label: Alpha Tom Kodamullil
+
```

Comment:
A GitHub handle for the contact is now requested in [FP-011](http://www.obofoundry.org/principles/fp-011-locus-of-authority.html) using the `github` key

Reply:
more specifically:
```yaml
contact:
  email: alpha.tom.kodamullil@scai.fraunhofer.de
  label: Alpha Tom Kodamullil
  github: <handle goes here>
```